examples/server/server.cpp

```diff
@@ -3519,6 +3767,12 @@ int main(int argc, char ** argv) {
     svr->Post("/v1/embeddings",       handle_embeddings);
     svr->Post("/tokenize",            handle_tokenize);
     svr->Post("/detokenize",          handle_detokenize);
+    if (!sparams.slot_save_path.empty()) {
+        // only enable slot endpoints if slot_save_path is set
+        svr->Post("/slot/save", handle_slot_save);
```

Comment:
To be restful compliant, the good pass would be `/slots/{slot-id}?action=save`, verb must not be present in the path and resources are in the plural form.

Reply:
Alright, I'll look into adding server test cases. Adjusting the endpoints as well if that's the preferred style.